Detectors/TPC/simulation/include/TPCSimulation/Point.h

```diff
@@ -175,18 +173,18 @@ class LinkableHitGroup : public o2::BaseHit {
 #ifdef HIT_AOS
   std::vector<o2::TPC::ElementalHit> mHits; // the hits for this group
 #else
-  std::vector<float> mHitsXVctr; //! do not stream this (just for memory handling convenience)
-  std::vector<float> mHitsYVctr; //! do not stream this
-  std::vector<float> mHitsZVctr; //! do not stream this
-  std::vector<float> mHitsTVctr; //! do not stream this
-  std::vector<short> mHitsEVctr; //! do not stream this
+  std::vector<float> mHitsXVctr; // do not stream this (just for memory handling convenience)
+  std::vector<float> mHitsYVctr; // do not stream this
+  std::vector<float> mHitsZVctr; // do not stream this
+  std::vector<float> mHitsTVctr; // do not stream this
+  std::vector<short> mHitsEVctr; // do not stream this
```

Comment:
Why was the exclamation mark removed? Should these data member be streamable or not? Does the class need to be streamable anyhow? if not you should set the version in the classdef to '0'. In any case, the Doxygen style comment requires a '<' So either '///<' or '//!<' (latter for not streamable).

Reply:
Well spotted. I had to play a bit here because the old solution did not work with std::vector out of the box. Will clean this up.